src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -2753,9 +2753,18 @@ def list_consumption_locations(cmd):
 
 
 def list_locations(cmd, sku, linux_workers_enabled=None):
-    client = web_client_factory(cmd.cli_ctx)
+    web_client = web_client_factory(cmd.cli_ctx)
```

Comment:
You can make `client` in parameter list in the function, which will utilize the client defined in your command.